wordpool/listgen/__init__.py

```diff
@@ -1,55 +1,71 @@
 """List generation and I/O."""
 
-from __future__ import print_function
-
-import random
-import os.path as osp
-import itertools
-
-import numpy.random as npr
-import pandas as pd
-
-from .. import load
-from .. import exc
-from . import fr
-from . import catfr
-from . import pal
-
-RAM_LIST_EN = load("ram_wordpool_en.txt")
-RAM_LIST_SP = load("ram_wordpool_sp.txt")
-
-CAT_LIST_EN = load("ram_categorized_en.txt")
-CAT_LIST_SP = load("ram_categorized_sp.txt")
-
-PRACTICE_LIST_EN = load("practice_en.txt")
-PRACTICE_LIST_SP = load("practice_sp.txt")
-
-LURES_LIST_EN = load("REC1_lures_en.txt")
+try:
+    import random
```

Comment:
Do you really need to do this for stdlib modules? I would have thought IronPython had these.

You should only be checking for `ImportError` for things like pandas which you know won't work in C#.